pkg/monitor/cluster/clusterwideproxystatus.go

```diff
@@ -162,7 +162,8 @@ func (mon *Monitor) emitCWPStatus(ctx context.Context) error {
 			return err
 		}
 		clusterDomain := clusterdetails.Spec.Domain
-		if !(noProxyMap[clusterDomain] || noProxyMap[".apps."+clusterDomain] || noProxyMap["."+clusterDomain]) {
+		clusterDomaincheck := noProxyMap[clusterDomain] || noProxyMap["."+clusterDomain]
+		if !(noProxyMap[".apps."+clusterDomain] || clusterDomaincheck) {
```

Comment:
Instead of adding the `clusterDomaincheck` individually, we should take it out as one if condition and then put the things that are to be executed in its absence inside it. Something like : 

```
if !(clusterDomaincheck) {
      //Perform other checks
}
```